react/src/components/flag/SprkFlag.test.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import Enzyme, { shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import SprkFlag from './SprkFlag';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('SprkFlag:', () => {
+  it('should display default styles', () => {
+    const wrapper = shallow(
+      <SprkFlag
+        media={
+          <img
+              alt="Provide useful alternative text"
+              src="https://spark-assets.netlify.com/spark-logo-mark.svg"
+          />
+        }
+      />
+    );
+    expect(wrapper.hasClass('sprk-o-Flag')).toBe(true);
+  });
+
+  it('should display reversed styles', () => {
+    const wrapper = shallow(
+      <SprkFlag
+        isReversed
+        media={
+          <img
+              alt="Provide useful alternative text"
+              src="https://spark-assets.netlify.com/spark-logo-mark.svg"
+          />
+        }
+      />
+    );
+    expect(wrapper.hasClass('sprk-o-Flag--rev')).toBe(true);
+  });
+
+  it('should display stacked style', () => {
+    const wrapper = shallow(
+      <SprkFlag
+        isStacked
+        media={
+          <img
+              alt="Provide useful alternative text"
+              src="https://spark-assets.netlify.com/spark-logo-mark.svg"
+          />
+        }
+      />
+    );
+    expect(wrapper.hasClass('sprk-o-Flag--stacked')).toBe(true);
+  });
+
+  it('should apply additional media classes', () => {
+    const testClass = "testMedia";
+    const wrapper = shallow(
+      <SprkFlag
+        additionalMediaClasses={testClass}
+        media={
+          <img
+              alt="Provide useful alternative text"
+              src="https://spark-assets.netlify.com/spark-logo-mark.svg"
+          />
+        }
+        additionalClasses={testClass}
+      />
+    );
+    expect(wrapper.hasClass(testClass)).toBe(true);
+  });
+
+  it('should apply additional body classes', () => {
+    const testClass = "testBody";
+    const wrapper = shallow(
+      <SprkFlag
+        additionalBodyClasses={testClass}
+        media={
+          <img
+              alt="Provide useful alternative text"
+              src="https://spark-assets.netlify.com/spark-logo-mark.svg"
+          />
+        }
+        additionalClasses={testClass}
+      />
+    );
+    expect(wrapper.hasClass(testClass)).toBe(true);
```

Comment:
This is checking if the test classes are on the main wrapper and not if they are on the body. Will have to to do wrapper.find(...body element...) and check the class on there. Right now this test is passing because you are also adding `additionalClasses={testClass}` to the main SprkFlag